pkg/api/http/directmessaging.go

```diff
@@ -100,7 +101,7 @@ func (a *api) onDirectMessage(w http.ResponseWriter, r *http.Request) {
 
 	targetID, invokeMethodName := findTargetIDAndMethod(reqPath, r.Header)
 	if targetID == "" {
-		respondWithError(w, messages.ErrDirectInvokeNoAppID)
+		respondWithError(w, apierrors.Invoke("invoke").WithAppError("", nil).NoAppID())
```

Comment:
@elena-kolevska Thank you for reviewing the PR and for the suggestions. I just addressed most of the suggestions and have run `gofumpt`. I executed the `gofumpt -w .` command in the dapr parent directory, because of which many files have been formatted. I hope it's a good thing.

Reply:
Can we keep a more broad formatting to a separate PR pls? It makes it more difficult to review and find the code that actually changed. 🙏